CoreUtilities/src/au/gov/asd/tac/constellation/utilities/csv/SmartCSVWriter.java

```diff
@@ -27,7 +28,14 @@
  * @author sirius
  */
 public class SmartCSVWriter implements AutoCloseable {
-
+    private static final Pattern FIND_QUOTES = Pattern.compile("\"");
+    
+    private static final char NEW_LINE = '\n';
```

Comment:
For consistency you can use SeparatorConstants.NEWLINE
Similarly, it might be worth adding line feed and other constants to that file